Update README.md (#4615)
returntocorp/semgrep: README.md

- Semgrep - the open-source command line tool at the heart of everything (this project)
- [Semgrep CI](https://semgrep.dev/docs/semgrep-ci/) - a specialized Docker image for running Semgrep in CI environments

> - [Semgrep Playground](https://semgrep.dev/editor) - an online interactive tool for writing and sharing rules

Reviewer: Why this change? Editor is more descriptive than tool, even if it's no longer the primary name

> - [Semgrep Playground](https://semgrep.dev/editor) - an online interactive tool for writing and sharing rules

Author: ```suggestion
- [Semgrep Playground](https://semgrep.dev/editor) - an online interactive rule builder for writing and sharing rules
```

# Check for Python == where the left and right hand sides are the same (often a bug)
$ semgrep -e '$X == $X' --lang=py path/to/src

> # Fetch rules automatically by examining your code's makeup (your source code

Author: ```suggestion
# Fetch rules automatically by setting the `--config auto` flag. 
# This will fetch rules relevant to your project from Semgrep Registry.
# The name of your project will be sent to Semgrep Registry as an identifier 
# to make selecting relevant rules fast next time; source code will not be uploaded.
```

# Check for Python == where the left and right hand sides are the same (often a bug)
$ semgrep -e '$X == $X' --lang=py path/to/src

# Fetch rules automatically by setting the `--config auto` flag. 
# This will fetch rules relevant to your project from Semgrep Registry.
# The name of your project will be sent to Semgrep Registry as an identifier 

> # to make selecting relevant rules fast next time; source code will not be uploaded.

Author: ```suggestion
# to make selecting relevant rules fast next time;
# source code will not be uploaded.
```

> # is not uploaded anywhere)

Author: ```suggestion
```
